pythonFiles/pytest-vscode-integration/tests/test_vscode_integration.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+
+
+def test_bar_fixture(testdir):
+    """Make sure that pytest accepts our fixture."""
+
+    # create a temporary pytest test module
+    testdir.makepyfile(
+        """
+        def test_sth(bar):
+            assert bar == "europython2015"
+    """
+    )
```

Comment:
Please run the code through Black (and make sure CI is checking for Black formatting).

```suggestion
    testdir.makepyfile("""
        def test_sth(bar):
            assert bar == "europython2015"
""")
```